crates/ruff_python_ast/generate.py

```diff
@@ -14,6 +14,16 @@
 
 import tomllib
 
+imports = ["crate::name::Name"]
+
+
+def requires_crate_prefix(ty: str) -> bool:
+    no_crate_prefix_types = [
+        "Name",  # Imported at the top
+        "bool",
+    ]
+    return not (ty in no_crate_prefix_types or ty.startswith("Box"))
```

Comment:
Come to think of it, once we have migrated _all_ of the node types over to this mechanism, none of them will need `crate::` qualifiers, since everything will be either (a) defined in the generated file, (b) in an `use` statement at the top of the generated file, or (c) a primitive of some kind.  So it might make sense to invert this into a list of types that _do_ need a crate prefix.  That would also eliminate the `Box` special case.

Reply:
Good idea, also while doing that I realized a lot of the `create::` that was added before was unnecessary. Also the list has a few elements we can get rid of soon after moving other things to generated.